python-sdk/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[build-system]
+requires = ["maturin>=1.7,<2.0"]
+build-backend = "maturin"
+
+[project]
+name = "eppo-server-sdk"
+description = "Eppo SDK for Python"
+readme = "README.md"
+authors = [{ name = "Eppo", email = "eppo-team@geteppo.com" }]
+license = { file = "LICENSE" }
+requires-python = ">=3.8"
```

Comment:
☝️ bumping required python version from 3.6 to 3.8

Reply:
Sounds good, given 3.6 hit end of life two years ago 😬